code/game/objects/items/weapons/storage/uplink_kits.dm

```diff
@@ -1,85 +1,85 @@
-				new /obj/item/weapon/card/id/syndicate(src)
+				new /obj/item/ammo_box/magazine/m10mm/hp(src)
+				new /obj/item/device/encryptionkey/syndicate(src)
+				new /obj/item/weapon/storage/fancy/cigarettes/cigpack_syndicate(src)
+				new /obj/item/weapon/implanter/krav_maga(src)
+				new /obj/item/weapon/reagent_containers/food/drinks/drinkingglass/alliescocktail(src)
+				return
 
-			if("screwed")
-				new /obj/item/device/radio/beacon/syndicate/bomb(src)
-				new /obj/item/weapon/grenade/syndieminibomb(src)
+			if("sabotage")
 				new /obj/item/device/powersink(src)
-				new /obj/item/clothing/suit/space/syndicate/black/red(src)
-				new /obj/item/clothing/head/helmet/space/syndicate/black/red(src)
+				new /obj/item/weapon/grenade/syndieminibomb(src)
+				new /obj/item/weapon/card/emag(src)
+				new /obj/item/weapon/grenade/clusterbuster/n2o(src)
+				new /obj/item/weapon/storage/box/syndie_kit/space(src)
+				new /obj/item/clothing/gloves/color/yellow(src)
+				new /obj/item/weapon/rcd/preloaded(src)
 				return
 
-			if("guns")
+			if("payday")
 				new /obj/item/weapon/gun/projectile/revolver(src)
 				new /obj/item/ammo_box/a357(src)
 				new /obj/item/weapon/card/emag(src)
-				new /obj/item/weapon/grenade/plastic/c4(src)
+				new /obj/item/weapon/grenade/plastic/x4(src)
+				new /obj/item/weapon/card/id/syndicate(src)
 				new /obj/item/clothing/gloves/color/latex/nitrile(src)
 				new /obj/item/clothing/mask/gas/clown_hat(src)
-				new /obj/item/clothing/under/suit_jacket/really_black(src)
-				return
-
-			if("murder")
-				new /obj/item/weapon/melee/energy/sword/saber(src)
-				new /obj/item/clothing/glasses/thermal/syndi(src)
-				new /obj/item/weapon/card/emag(src)
-				new /obj/item/clothing/shoes/syndigaloshes(src)
 				return
 
 			if("implant")
-				new /obj/item/weapon/implanter/freedom(src)
-				new /obj/item/weapon/implanter/uplink(src)
-				new /obj/item/weapon/implanter/emp(src)
 				new /obj/item/weapon/implanter/adrenalin(src)
-				new /obj/item/weapon/implanter/explosive(src)
 				new /obj/item/weapon/implanter/storage(src)
+				new /obj/item/weapon/implanter/freedom(src)
+				new /obj/item/weapon/implanter/emp(src)
+				new /obj/item/organ/internal/cyberimp/eyes/thermals(src)
 				return
 
 			if("hacker")
 				new /obj/item/weapon/aiModule/syndicate(src)
-				new /obj/item/weapon/card/emag(src)
 				new /obj/item/device/encryptionkey/binary(src)
 				new /obj/item/weapon/aiModule/toyAI(src)
 				return
 
-			if("lordsingulo")
-				new /obj/item/device/radio/beacon/syndicate(src)
-				new /obj/item/clothing/suit/space/syndicate/black/red(src)
-				new /obj/item/clothing/head/helmet/space/syndicate/black/red(src)
-				new /obj/item/weapon/card/emag(src)
-				return
-
 			if("darklord")
-				new /obj/item/weapon/melee/energy/sword/saber(src)
-				new /obj/item/weapon/melee/energy/sword/saber(src)
+				new /obj/item/weapon/melee/energy/sword/saber/red(src)
+				new /obj/item/weapon/melee/energy/sword/saber/red(src)
```

Comment:
As a matter of fact, no... I just assumed they would, but in testing it still comes up random. Maybe I should look into making that a thing.